server/graphql/common/scope-check.ts

```diff
@@ -116,11 +117,27 @@ export const checkRemoteUserCanRoot = (req: express.Request): void => {
 type OAuthScope = keyof typeof OAuthScopes;
 
 export const checkScope = (req: express.Request, scope: OAuthScope): boolean => {
-  return !req.userToken || req.userToken.hasScope(scope);
+  if (req.userToken) {
+    return req.userToken.hasScope(scope);
+  } else if (req.personalToken) {
+    // Personal Tokens had no scope until this date, all scopes were assumed
+    if (moment('2023-01-23') > moment(req.personalToken.createdAt)) {
```

Comment:
2023-01-23

Reply:
updatedAt ?

Reply:
Alternative: update all personal tokens created before `2023-01-23` to give them all scopes. Should be followed by a deprecation policy (users must update the tokens or they'll be removed)